src/components/views/settings/ProfileSettings.js

```diff
@@ -27,27 +27,32 @@ export default class ProfileSettings extends React.Component {
         super();
 
         const client = MatrixClientPeg.get();
-        let user = client.getUser(client.getUserId());
-        if (!user) {
-            // XXX: We shouldn't have to do this.
-            // There seems to be a condition where the User object won't exist until a room
-            // exists on the account. To work around this, we'll just create a temporary User
-            // and use that.
-            console.warn("User object not found - creating one for ProfileSettings");
-            user = new User(client.getUserId());
-        }
-        let avatarUrl = user.avatarUrl;
-        if (avatarUrl) avatarUrl = client.mxcUrlToHttp(avatarUrl, 96, 96, 'crop', false);
         this.state = {
             userId: user.userId,
-            originalDisplayName: user.displayName,
-            displayName: user.displayName,
-            originalAvatarUrl: avatarUrl,
-            avatarUrl: avatarUrl,
+            originalDisplayName: undefined,
+            displayName: undefined,
+            originalAvatarUrl: undefined,
+            avatarUrl: undefined,
```

Comment:
we typically use `null` instead

Reply:
[Except when not?](https://github.com/matrix-org/matrix-react-sdk/pull/3739/files/370b7d45119b35bfc087582338d67946c89c2dc4#diff-d23966825df49bff164e0186b877ad39R67)